frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -103,14 +103,26 @@ function getTableRowForCategory(category: StatusCategory, polling_station: Polli
     );
   }
 
+  function formatFinishedAt(date: Date) {
+    const today = new Date();
+    if (
+      date.getDate() === today.getDate() &&
+      date.getMonth() === today.getMonth() &&
+      date.getFullYear() === today.getFullYear()
+    ) {
+      return `${t("today")} ${date.toLocaleTimeString("nl-NL", { timeStyle: "short" })}`;
+    } else {
+      return date.toLocaleString("nl-NL", {
```

Comment:
after some SQL magic:
![image](https://github.com/user-attachments/assets/cb0ecf6f-659d-43ab-ae3b-fc3a543c7572)

Reply:
Might get confusing if the data is a week old:
![image](https://github.com/user-attachments/assets/4b4d7bba-e468-4111-a307-24ea49b5792c)

Reply:
discussed this before right? Chance data is > 1 week old is small. But if we want to prevent this, we could revert to showing the date in short format (i.e. 4 dec 13:42) if the day is more then 6 days before today...

Reply:
Oh god another if statement, but definitely doable, I'll implement the above comment ^^

Reply:
@jorisleker @jschuurk-kr Please review again, I implemented your suggestions in bfd3dc8e110b265709b21affe8630fba3d1551e9

Reply:
@jschuurk-kr I cannot really test the function I added now. Shall I move it to a util file and test it there?